managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorContainer.java

```diff
@@ -124,6 +130,8 @@ public void removeCursor(String name) {
 
             PositionImpl previousSlowestConsumer = heap.get(0).position;
 
+            if (item.cursor.isDurable()) {
+
```

Comment:
nit: the section below should get indented.

Reply:
fixed